AstronomyObjC/AstronomyObjC/Controllers/TMCNetworkController.m

```diff
@@ -0,0 +1,90 @@
+//
+//  TMCNetworkController.m
+//  AstronomyObjC
+//
+//  Created by Christopher Devito on 5/18/20.
+//  Copyright © 2020 Christopher Devito. All rights reserved.
+//
+
+#import "TMCNetworkController.h"
+#import "AstronomyObjC-Swift.h"
+#import "NSObject+NSJSONSerialization.h"
+#import "TMCMarsPhotoReference.h"
+#import "NSURL+TMCUsingHTTPS.h"
+
+@implementation TMCNetworkController
+
+static NSString *apiKey = @"IjCchzjn0EuNgmVSr824DPsW57IXpXMwg7bS9mnc";
+
+- (instancetype)initWithPhotos:(NSMutableArray *)photos
+{
+    self = [super init];
+    if (self) {
+        _photos = photos;
+    }
+    return self;
+}
+
+- (void)fetchMarsPhotosOnSol:(NSNumber *)sol
+             completionBlock:(TMCMarsPhotoCompletion)completionBlock {
+
+    NSURL *baseURL = [NSURL URLWithString:@"https://api.nasa.gov/mars-photos/api/v1/rovers/curiosity/photos"];
+    NSURLComponents *components = [NSURLComponents componentsWithURL:baseURL resolvingAgainstBaseURL:YES];
+    NSURLQueryItem *solTerm = [NSURLQueryItem queryItemWithName:@"sol" value:[NSString stringWithFormat:@"%i", sol.intValue]];
+    NSURLQueryItem *page = [NSURLQueryItem queryItemWithName:@"page" value:[NSString stringWithFormat:@"%i", 1]];
```

Comment:
Great work paging the data!

It would be nice to see more pages as you scroll.